apps/OboeTester/app/src/main/java/com/mobileer/oboetester/CommunicationDeviceView.java

```diff
@@ -122,14 +151,41 @@ private void setSpeakerPhoneOn(boolean enabled) {
 
     private void showCommDeviceStatus() {
         boolean enabled = mAudioManager.isSpeakerphoneOn();
-        String text = (enabled ? "ON" : "OFF");
+        String text = ":" + (enabled ? "ON" : "OFF");
         if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.S) {
             AudioDeviceInfo commDeviceInfo = mAudioManager.getCommunicationDevice();
             if (commDeviceInfo != null) {
-                text += ", CommDev=" + commDeviceInfo.getId();
+                text += ", #" + commDeviceInfo.getId();
             }
         }
-        mIsSpeakerphoneText.setText(" => " + text);
+        mSpeakerStatusView.setText(text + ",");
+
+        if (mScoState == AudioManager.SCO_AUDIO_STATE_CONNECTING) {
+            text = ":wait";
```

Comment:
nit: wait is in a different font as CON and DISCO

Reply:
Also change to DISCON